src/librustdoc/html/static/css/rustdoc.css

```
@@ -1046,7 +1046,7 @@ so that we can apply CSS-filters to change the arrow color in themes */
 	position: absolute;
 	top: 100%;
 	right: 0;
-	z-index: 2;
+	z-index: 101;
```

I know that we haven't done it like that for other z-indices but personally speaking I'd love to use CSS variables here to make this more robust. Something like

```suggestion
	z-index: calc(var(--sidebar-z-index) + 1);
```

where we have defined `--sidebar-z-index` somewhere reasonable. Not sure what the other rustdoc reviewers think about that.

---

It'd even be self-documenting, too, that's a bonus.